battle.go

```diff
@@ -277,6 +281,19 @@ func (b *Battle) SimulateRound() ([]Transaction, bool) {
 				Item:   t.Item,
 				Move:   t.Target.Pokemon.Moves[t.Move],
 			})
+		case SwitchTurn:
+			party := b.GetParty(&t.Target)
+			if _, ok := party.activePokemon[t.Target.partySlot]; ok {
+				blog.Panic(ErrorCannotSwitch)
```

Comment:
We _really_ should stop panicking to throw errors. Definitely a job for another PR though.